src/modules/person/persistence/person.scope.integration-spec.ts

```diff
@@ -12,10 +12,13 @@ import {
 } from '../../../../test/utils/index.js';
 import { ScopeOrder } from '../../../shared/persistence/scope.enums.js';
 import { PersonDo } from '../domain/person.do.js';
+import { OrganisationDo } from '../../organisation/domain/organisation.do.js';
 import { PersonPersistenceMapperProfile } from './person-persistence.mapper.profile.js';
 import { PersonEntity } from './person.entity.js';
 import { PersonScope } from './person.scope.js';
 import { faker } from '@faker-js/faker';
+import { PersonenkontextDo } from '../../personenkontext/domain/personenkontext.do.js';
+import { PersonenkontextEntity } from '../../personenkontext/persistence/personenkontext.entity.js';
 
 describe('PersonScope', () => {
```

Comment:
Wo ist der findByRolleId Test?

Reply:
in the integration test i did not do a Rolle test only in unit test